features/distro/pom.xml

```diff
@@ -23,22 +23,15 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>org.eclipse.smarthome</groupId>
-            <artifactId>esh-tp</artifactId>
-            <version>${esh.version}</version>
-            <classifier>features</classifier>
-            <type>xml</type>
-        </dependency>
-        <dependency>
-            <groupId>org.eclipse.smarthome</groupId>
-            <artifactId>esh-core</artifactId>
-            <version>${esh.version}</version>
+            <groupId>org.openhab.core.features.karaf</groupId>
```

Comment:
@maggu2810 Once again, I have to ask for your help on some Karaf issue:

As you can see [in the PR build](https://ci.openhab.org/job/PR-openHAB-Distribution/52/console), when creating the distro feature the versions for jna and jetty seem to be lost (which are defined in the parent pom(s)). When I remove the dependency to the new `org.openhab.core.features.karaf.openhab-core` feature here, everything works (as before). I have no clue, why this dependency can make such a difference...
There are also warnings in the log:
```
[WARNING] Manifest not present in the first entry of the zip - org.openhab.core.bom.openhab-core-2.5.0-SNAPSHOT.pom
[WARNING] Manifest not present in the first entry of the zip - org.openhab.core.bom.test-2.5.0-SNAPSHOT.pom
```
Maybe this is somehow related?

Reply:
Ah - forget it, I found the issue! PR for openhab-core is coming...

Reply:
Sorry, same here. Didn't see that notification.

Reply:
No worries, you see that I was able to help myself :-)